### ISSUE TITLE
(A) Unable to view all graph data points with keyboard

### ISSUE THREAD

**Issue by otis-bath** [open-sdg/open-sdg#1756]
**High level (A)**

On this example (15-2-1) if you use the keyboard to focus on the graph you are unable to view all the data points, this is likely to be occuring on other examples too. The data points after 2011 are not keyboard focusable. This means that not keyboard only users cannot read all of the information is the chart.

![image](https://user-images.githubusercontent.com/90769108/182147096-c57cab20-ce5d-45dd-8cc6-3d42d56602e4.png)

![image](https://user-images.githubusercontent.com/90769108/182146988-5a780777-5dc2-468e-a8e3-6039cd1ca680.png)

The solution should ensure that it doesn't 'cycle' back to the initial data like this currently does.
